### ISSUE TITLE
Does the Acc in train.log means the rank-1 acc of the train dataset?

### ISSUE THREAD

**Issue by WannaSeaU** [KaiyangZhou/deep-person-reid#166]

**Comment by KaiyangZhou** [comment]
They are not exactly the same because they refer to different metrics, one is classification while the other is retrieval. They are measured in different ways, i.e. logits vs. feature distance.
